src/main/java/com/esri/samples/ogc/open_street_map_layer/README.md

```diff
@@ -0,0 +1,18 @@
+# OpenStreetMap Layer
+Add the OpenStreetMap layer to a map as a Basemap.
+
+![OpenStreetMap Layer App](openstreetmap-layer.png)
+

```

Comment:
This file needs to be in html while still having the .md extension. The reason is the sample viewer renders these in a webview as html. Look at the raw text of some of the other sample readmes to get an idea of the format.